recaptcha_enterprise/snippets/test_create_assessment.py

```diff
@@ -137,3 +159,21 @@ def assess_token(recaptcha_site_key: str, token: str, action: str) -> Assessment
 
 def set_score(browser: WebDriver, score: str) -> None:
     browser.find_element(By.CSS_SELECTOR, "#assessment").send_keys(score)
+
+
+def get_hashed_account_id() -> str:
```

Comment:
```suggestion
def _get_hashed_account_id() -> str:
```
If it's an internal helper method, prepend `_`.